src/cli/src/export.rs

```diff
@@ -56,6 +56,11 @@ pub struct ExportCommand {
 
     /// Directory to put the exported data. E.g.: /tmp/greptimedb-export
     /// for local export.
+    ///
+    /// if both `output_dir` and `s3` are set, `output_dir` will be only used for
+    /// exported SQL files, and the data will be exported to s3.
+    ///
+    /// if `s3` is set but `output_dir` is not set, both SQL&data will be exported to s3.
```

Comment:
If i understand correctly, the proper name must be `ddl_dir`?

Reply:
> If i understand correctly, the proper name must be `ddl_dir`?

added a `s3_ddl_local_dir` option, this verbose name is to prevent misunderstanding(i.e. misunderstand as s3 path or else)